docker/luna_tutorial/README.md

```diff
@@ -8,16 +15,13 @@ $ make help [ARGS]
 clean                cleanup luna-tutorial images and containers
 clean-image          remove luna tutorial image
 build                build containers
-run                  launch containers. Default IP:PORT=127.0.0.1:8888. Optionally specify args HOST_IP=xxx.xxx.xxx.xxx HOST_PORT=xxxx.
+run                  launch containers
 stop                 terminate containers but keep volumes
 exec                 launches terminal prompt inside luna_tutorial container
 
 ```
 
-In order to run this tutorial, first build the docker-compose image and then lauch the containers. Once the containers are launched, you may view the various interfaces at these urls. 
-
-    jupyterlab: http://localhost:8888  # NOTE insert token from vmount/logs/tutorial.log 
-    DSA:        http://localhost:8080
+In order to run this tutorial, first build the docker-compose images using `make build` and then lauch the containers using `make run`. Once the containers are launched, copy-paste the URL from the `make run` command into a browser and enter the token from `vmount/logs/tutorial.log`.  
```

Comment:
We can probably remove `make build` as build is done when running `docker-compose up`.